packages/tessellate-bundler/package.json

```diff
@@ -19,54 +19,54 @@
     "url": "https://github.com/zalando-incubator/tessellate.git"
   },
   "scripts": {
-    "dist": "webpack",
+    "clean": "rimraf dist/*",
+    "dist": "babel -d dist src",
```

Comment:
Webpack is not really a good choice for server-side node projects. It also just uses babel internally but compiles everything into one file and adds some cruft. Using babel directly is the cleaner approach.

We used webpack in [Zappr](https://github.com/zalando/zappr) so that's why I started tessellate with it as well. Now I realize it was a bad idea.